src/main/java/xyz/groundx/caver_ext_kas/wallet/KASWallet.java

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright 2020 The caver-java-ext-kas Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the “License”);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package xyz.groundx.caver_ext_kas.wallet;
+
+import com.klaytn.caver.account.AccountKeyRoleBased;
+import com.klaytn.caver.account.AccountKeyWeightedMultiSig;
+import com.klaytn.caver.methods.response.AccountKey;
+import com.klaytn.caver.transaction.AbstractFeeDelegatedTransaction;
+import com.klaytn.caver.transaction.AbstractFeeDelegatedWithRatioTransaction;
+import com.klaytn.caver.transaction.AbstractTransaction;
+import com.klaytn.caver.transaction.TransactionDecoder;
+import com.klaytn.caver.transaction.type.AccountUpdate;
+import com.klaytn.caver.transaction.type.LegacyTransaction;
+import com.klaytn.caver.wallet.IWallet;
+import com.klaytn.caver.wallet.keyring.SignatureData;
+import xyz.groundx.caver_ext_kas.kas.wallet.Wallet;
+import xyz.groundx.caver_ext_kas.rest_client.io.swagger.client.ApiException;
+import xyz.groundx.caver_ext_kas.rest_client.io.swagger.client.api.wallet.model.*;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Representing a wallet class that uses KAS Wallet API.<br>
+ * KASWallet used as a member of CaverExtKAS named wallet.<br>
+ * If you called sign() to sign Caver's Transaction(ValueTransfer, SmartContractExecution..), It is signed using KAS Wallet API internally.
+ */
+public class KASWallet implements IWallet {
+    /**
+     * The WalletAPI instance to use KAS Wallet API.
+     */
+    Wallet walletAPI;
+
+    /**
+     * Creates a wallet instance that uses the KAS Wallet API. <br>
+     * KASWallet used as a member of CaverExtKAS named wallet.
+     * @param walletAPI An WalletAPI instance to use KAS Wallet API.
+     */
+    public KASWallet(Wallet walletAPI) {
+        this.walletAPI = walletAPI;
+    }
+
+    /**
+     * Generates accounts in KAS Wallet service.
+     * @param num The number of accounts to generate.
+     * @return List
+     */
+    @Override
+    public List<String> generate(int num) throws ApiException {
```

Comment:
Can you implement `public List<String> generate() throws ApiException` additionally??